sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-verifyAssessmentCalculationOrder.e2e-spec.ts

```diff
@@ -1156,6 +1160,8 @@ describe("AssessmentController(e2e)-verifyAssessmentCalculationOrder", () => {
               exemptScholarshipsBursaries: 2000,
               totalFederalFSC: 3000,
               totalProvincialFSC: 4000,
+              returnTransportationCost: 500,
+              BooksCost: 600,
```

Comment:
Please start the property name with lower letter.

Reply:
I already fixed that. Thanks.